File: Aclass/A1005.cpp
#include <iostream>
#include <cstring>
using namespace std;
string num[10]={"zero","one","two","three","four","five","six","seven","eight","nine"};
int main()
{
	string snum;
	cin>>snum;
	int sum=0,len=snum.length();
	for(int i=0;i<len;i++)
		sum+=snum[i]-'0';
	string sum_str=to_string(sum);
	int len2=sum_str.length();
	for(int i=0;i<len2;i++)
	{
		cout<<num[sum_str[i]-'0'];
		if(i!=len2-1)
			cout<<" ";
	}
	return 0;
}

File: Aclass_Python/A1005.py
num_in_English = ['zero', 'one', 'two', 'three', 'four', 'five', 'six', 'seven', 'eight', 'nine']
num = input()
sum = 0
ans = []
for i in range(len(num)):
    sum += int(num[i])
# 更简洁的方法： a = sum([int(i) for i in input()])
sum_string = str(sum)
for i in range(len(sum_string)):
    ans.append(num_in_English[int(sum_string[i])])
print(" ".join(ans))